### ISSUE TITLE
BodyParser 1.0.1 - ENOENT on install

### ISSUE THREAD

**Issue by MaarekElets** [allcount/allcountjs#14]
It appears that version 1.0.1 of BodyParser is no longer available as I am getting ENOENT errors on attempted install of both the allcountjs file (when it hit that dependency) AND when I try to install that version of BodyParser itself. I'm getting this from multiple machines on different networks (and different OSes). I can install the latest version of BodyParser, just not this or 1.0.2. 

This is on node 0.10.38 and NPM 2.9.0

**Comment by MaarekElets** [comment]
I should add that I also saw this error on Q as well. Never saw an issue with underscore though. Those appear to be the only static versioned dependencies in the package.json file.

**Comment by MaarekElets** [comment]
I'm closing this. I reinstalled nodejs (and symlinked to node) on my Debian server and now it's working. Same uninstall/reinstall worked on Windows as well. I have no idea what happened to cause it, but I'm not the only one on these servers working with node so it may have just been broken at some point.

**Comment by paveltiunov** [comment]
Hello Jeremy!
Nice to hear that you've resolved it. Have you installed allcountjs as global or as local dependency? Seems to be a failed install issue. Maybe `npm update` could help in such situation.
Thanks for describing your case!

**Comment by MaarekElets** [comment]
Oddly npm update didn't help me and I realized I the install had an issue when sudo npm came back saying it couldn't find the file (non elevated npm was fine) . The problem ended up being an attempted NVM node install, turns out they were having bower issues and didn't just do the symlink. In doing so both nodejs and npm packages were broken. That explains the Linux server issues but I still have no idea why I got the same error on Windows.... I'm going to chalk it up to losing track of vm windows and mistaking bash for git bash in Windows. I can't get it to repro for me anyway so I'd rather just plead PEBKAC.
